packit_service/worker/copr_build.py

```diff
@@ -287,6 +287,32 @@ def job_owner(self) -> Optional[str]:
 
         return self.api.copr_helper.copr_client.config.get("username")
 
+    def report_status_for_chroot(
+        self, description: str, state: str, url: str = "", chroot: str = ""
+    ):
+        self.report_status_to_build_chroot(
+            description, state, url, PRCheckName.get_build_check(chroot)
```

Comment:
Maybe move the `PRCheckName.get_build_check` to the `report_status_to_test_chroot` and use `chroot` as an input for all three methods. (For builds as well.)